tests/e2e/helper_test.go

```diff
@@ -36,6 +36,7 @@ const (
 	servicemeshNamespace     = "openshift-operators"
```

Comment:
not for this PR, but we should probably have a namespace per operator, since the default namespace suggested by specific operators may be different,  i.e. for serverless it is `openshift-serverless`

Reply:
Not related as well, but I am wondering why one of them toggles "Installing" (I looked the console) when if it's installed it should not change the configuration.

Reply:
>Not related as well, but I am wondering why one of them toggles "Installing" (I looked the console) when if it's installed it should not change the configuration.
so you already have operatro installed, then run e2e and  operator flipped from installed->installing->installed ?

Reply:
Yes, Succeed -> Installing -> Succeed. Not for you?

Reply:
Hmm, rechecked now, did not observe. Some glitch in old cluster may be.